bundle.Dockerfile

```diff
@@ -6,7 +6,7 @@ LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=ansibleee-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=alpha
-LABEL operators.operatorframework.io.metrics.builder=operator-sdk-v1.22.0
+LABEL operators.operatorframework.io.metrics.builder=operator-sdk-unknown
```

Comment:
This seems odd.  What version of operator SDK were you using?

Reply:
mmm ok I suspect the problem there is the fact I updated my system and it got the new sdk version:

```
> s pacman -Qi operator-sdk
Name            : operator-sdk
Version         : 1.25.0-1
Description     : SDK for building Kubernetes applications
Architecture    : x86_64
URL             : https://github.com/operator-framework/operator-sdk
```

I'll try to `make bundle` using the tool provided by [1].
Quick question: should this file (bundle.Dockerfile) be omitted here? It's generated by that command but I suspect it shouldn't be part of the PR (and generated in a different context).

[1] https://github.com/openstack-k8s-operators/install_yamls/blob/master/devsetup/roles/download_tools/tasks/main.yaml#L31-L48

Reply:
Yeah, we don't need to store that file in the repo.  It looks like maybe `.gitignore` is missing some content that we set in other operators, for instance: https://github.com/openstack-k8s-operators/cinder-operator/blob/60bdac1a3f2db7cd1ea1e8a43a3309bc3b4209e7/.gitignore#L20-L23

Reply:
ack thanks, I'm going to add that section in `.gitignore`